### PR TITLE
Add a simple Admin UI to manage systemtags

### PR DESCRIPTION
**Pull request by nickvergessen**
ToDo:
- [ ] API seems to support groups (?), maybe something for later/second PR

Ref 1. part of #231 

@mar1u5 in case you want to help testing

### CODE REVIEW

File: apps/systemtags/js/admin.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
+(function() {
```
Comment (nickvergessen) [inline apps/systemtags/js/admin.js]: Do we use the full file header in JS as well, or do we have a shorter second version?

File: apps/systemtags/templates/admin.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+vendor_script('core', 'select2/select2');
+vendor_style('core', 'select2/select2');
+script('core', [
+	'oc-backbone-webdav',
+	'systemtags/systemtags',
+	'systemtags/systemtagmodel',
+	'systemtags/systemtagscollection',
+]);
+
+script('systemtags_manager', 'admin');
```
Comment (icewind1991) [inline apps/systemtags/templates/admin.php]: appid should be `systemtags`

**Comment by icewind1991** [comment]
the ui seems very clunky and different to most other ui's we have for similar things.

imo having a list with all tags that allow inline editing would be a lot nices

(it would also probably make sense to have it in a seperate page/app entirely so the ui gets plenty of space)

**Comment by nickvergessen** [comment]
> imo having a list with all tags that allow inline editing would be a lot nices

That list might be too long

> it would also probably make sense to have it in a seperate page/app entirely so the ui gets plenty of space

The problem is the tags are delivered via webdav, so I was happy that it was already managed with backbone. I'd go with this simple approach for now (10), and we can still change it and come up with a better UI later on.

File: apps/systemtags/templates/admin.php (hunk 2 above)
Comment (nickvergessen) [inline apps/systemtags/templates/admin.php]: fixed

File: apps/systemtags/js/admin.js (hunk 1 above)
Comment (LukasReschke) [inline apps/systemtags/js/admin.js]: cc @schiessle

**Comment by mar1u5** [comment]
> (it would also probably make sense to have it in a seperate page/app entirely so the ui gets plenty of space)

Maybe we could solve that via #267 ... How did we do that in EE Workflow? @icewind1991

**Comment by nickvergessen** [comment]
> How did we do that in EE Workflow?

@mar1u5 similar to the current solution

**Comment by nickvergessen** [comment]
Rebased, please review

**Comment by MorrisJobke** [comment]
It's really rough from an UX point of view. But it works. Maybe we can sit together next week and refine the UI. 

👍 from me

**Comment by LukasReschke** [comment]
👍

**Comment by LukasReschke** [comment]
@nickvergessen Mind opening a backport to 10? :)

**Comment by nickvergessen** [comment]
Backport in https://github.com/nextcloud/server/pull/619

**Pull request merged**